crates/mun_test/src/fixture.rs

```diff
@@ -0,0 +1,221 @@
+use compiler::RelativePathBuf;
+use itertools::Itertools;
+
+const DEFAULT_FILE_NAME: &str = "main.mun";
+const META_LINE: &str = "//-";
+
+/// A `Fixtures` describes an single file in a project workspace. `Fixture`s can be parsed from a
```

Comment:
```suggestion
/// A `Fixture` describes a multiple files in a project workspace. `Fixture`s can be parsed from a
```

Reply:
No thats actually correct. The struct itself describes a single file in a project workspace. When calling parse a collection is returned.

Reply:
The fixtures (plural class name) part or the singular file? It seems like you define multiple Mun files in your fixture, not a single one?

Reply:
It should be `A Fixture describes a single file in a project workspace.` Does that make sense?

Reply:
I guess my understanding of what a fixture is, is wrong then. I thought that the following snippet contained one fixture, comprised of two files.

```rust
Fixture::parse(
    r#"
    //- /foo.mun
    fn hello_world() {
    }

    //- /bar.mun
    fn baz() {
    }
"#
)
```

Reply:
Parsing the given strings returns a `Vec<Fixture>`. One entry for `foo.mun` and one for `bar.mun`.